From 3beb971b10659cf78144ddc5eeea83501384440c Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Wed, 1 Aug 2018 11:16:30 -0400
Subject: [PATCH 30/55] Undo part of our old openssl version rollback.

When OpenSSL 1.1.0e didn't work so well, we added a macro for abort() to
passify the build.  Now that we've got 1.1.0e in again, that macro
messes up building SysCall/CrtWrapper.c.  This patch gets rid of the
macro.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 Cryptlib/Include/CrtLibSupport.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Cryptlib/Include/CrtLibSupport.h b/Cryptlib/Include/CrtLibSupport.h
index 76d7351..2eeab9a 100644
--- a/Cryptlib/Include/CrtLibSupport.h
+++ b/Cryptlib/Include/CrtLibSupport.h
@@ -380,7 +380,6 @@ extern FILE  *stdout;
 #define atoi(nptr)                        AsciiStrDecimalToUintn(nptr)
 #define gettimeofday(tvp,tz)              do { (tvp)->tv_sec = time(NULL); (tvp)->tv_usec = 0; } while (0)
 #define gmtime_r(timer,result)            (result = NULL)
-#define abort()
 
 void clear_ca_warning();
 BOOLEAN get_ca_warning();
-- 
2.17.1

